packages/nuxt/src/pages/module.ts

```diff
@@ -95,6 +95,8 @@ export default defineNuxtModule({
       return false
     }
     options.enabled = await isPagesEnabled()
+    // to be reviewed
+    nuxt.options.pages = options
```

Comment:
if we assign the whole object (with array of patterns), we might find that defu keeps inflating the array with duplicate patterns - we can deduplicate above.